.golangci.yml

```diff
@@ -13,16 +13,16 @@
 #
 
 run:
-  concurrency: 4
-  timeout: 5m
-  issues-exit-code: 0 # TODO: once all enabled linters are satisfied  
+  concurrency: 8
+  timeout: 2m
   tests: false
   skip-dirs:
     - ^experimental.*($|/)
```

Comment:
I know you didn't add the *experimental* directory to the skip list, but it should not be skipped, because it is considered when building `go.mod/go.sum`.